driver/Cargo.toml

```diff
@@ -5,6 +5,7 @@ authors = ["Alexander Herrmann <alex@gnosis.io>", "Ben Smith <ben@gnosis.io>", "
 edition = "2018"
 
 [dependencies]
+anyhow = "1"
```

Comment:
I have a question about this. I notice that we usually use exact versions, even for dependencies that are 1.0 . Why? Isnt just `1` enough?

Reply:
Looking at the json.lock, aren't we using anyhow 1.0.26?

Reply:
I think that means that its pulled in by a dependency but we are not using it since its not in Cargo.toml. But if there is no problem then I'll just keep the "anyhow = 1" .

Reply:
I'm a firm believer of `major.minor`. Note that they have different meanings:
- `1` <=> `^1.0.0` <=> `>=1.0.0 <2.0.0`
- `1.2` <=> `~1.2.0` <=> `>=1.2.0 <1.3.0`
- `1.2.3` <=> `~1.2.3` <=> `>=1.2.3 <1.3.0`

Reply:
Why specify the minor when semver guarantees that as long as the major is the same its compatible?